miniprofiler/context.go

```diff
@@ -0,0 +1,49 @@
+// +build go1.7
+
+package miniprofiler
+
+import (
+	"context"
+	"net/http"
+)
+
+// contextKey can be used to retreive the profiler instance from the request's context
+const contextKey = "MiniProfiler"
```

Comment:
I think we can use an unexported type here (https://twitter.com/Sajma/status/752476790304542720) so other users of context can't collide with this package.